sdk/dotnet/Addon.cs

```diff
@@ -11,6 +11,27 @@ namespace Pulumi.Pagerduty
 {
     /// <summary>
     /// With [add-ons](https://v2.developer.pagerduty.com/v2/page/api-reference#!/Add-ons/get_addons), third-party developers can write their own add-ons to PagerDuty's UI. Given a configuration containing a src parameter, that URL will be embedded in an iframe on a page that's available to users from a drop-down menu.
+    /// 
+    /// ## Example Usage
+    /// 
+    /// 
+    /// 
+    /// ```csharp
+    /// using Pulumi;
+    /// using Pagerduty = Pulumi.Pagerduty;
```

Comment:
Our casing of `PageDuty` is wrong too...

Reply:
Lets make a pass at these as a follow up